packages/web-components/@sfx/autocomplete/test/unit/autocomplete.test.ts

```diff
@@ -28,6 +29,14 @@ describe('Autcomplete Component', () => {
   });
 
   describe('connectedCallback', () => {
+    let windowAddEventListener;
+    let autocompleteAddEventListener;
+
+    beforeEach(() => {
+      windowAddEventListener = stub(window, 'addEventListener');
+      autocompleteAddEventListener = stub(autocomplete, 'addEventListener');
```

Comment:
This stub isn't being used.

Reply:
It's being used by the function under test, even if it's not being used explicitly here. The idea is to not be accidentally adding event listeners, which is what will happen if I remove the stub.

Reply:
YOU'RE RIGHT